README.md

```diff
@@ -107,292 +107,73 @@ Following the installation documentation below, you'll be able to run your own i
+You'll need to install the following packages :
 
-You'll need to install the following packages on your Operating System:
+    sudo apt install git realpath python3 python-virtualenv python3-dev python-pip build-essential libxml2-dev libxslt1-dev python-lxml zlib1g-dev redis-server
 
-    apt-get install git realpath python3 python-virtualenv python3-dev python-pip build-essential libxml2-dev libxslt1-dev python-lxml
+## Get the source ##
 
-You also need `redis-server` package if you want to send email in async mode.
+Get the sources from GitHub:
 
-## Database ##
+    git clone https://github.com/tracim/tracim.git
+    cd tracim/
 
-If you want use PostgreSQL as database engine:
+*Note: Now everything is documented to be executed from the tracim directory newly created.*
 
-    apt-get install postgresql-server-dev-all postgresql postgresql-client
+## Frontend dependencies ##
 
-Or if you want to use MySQL as database engine
+[//]: # ( from https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)
 
-    apt-get install mysql-server mysql-client libmysqlclient-dev
+Install `nodejs` by typing:
 
-Or if you want to use SQLite as database engine
+    curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+    sudo apt install -y nodejs
 
-    apt-get install sqlite3
+Check that this went well by getting `npm` version:
 
-## Installation ##
+    npm -v
 
-### Get the source ###
+Then install frontend dependencies listed in the file `package.json`:
 
-Get the sources from github with git:
+    npm install
 
-    git clone https://github.com/tracim/tracim.git
-    cd tracim/
-
-*Note: Now everything is documented to be executed from the tracim directory newly created.*
+At last, compile frontend files:
 
-### Setting-up python virtualenv ###
+    npm run gulp-dev # for a development environment
+    # npm run gulp-prod # for a production environment
 
-_Reminder : Tracim is developed and tested using python3.4._
+## Tracim virtual environment ##
 
-We strongly recommend to use virtualenv as deployment environment. This ensure that there will be no conflict between system-wide python installation and Tracim required ones. To Create the virtual environment:
+Create a python virtual environment:
 
     virtualenv -p /usr/bin/python3.4 tg2env
 
-And to activate it in your terminal session (**all tracim command execution must be executed under this virtual environment**)):
+Activate it in your terminal session (**all tracim command execution must be executed under this virtual environment**):
 
     source tg2env/bin/activate
 
-To install tracim and it's dependencies:
+Install Tracim and its dependencies:
 
     cd tracim && python setup.py develop && cd -
     pip install -r install/requirements.txt
 
-**Note**: If you want to use MySQL database, please refer to Configuration/database schema note to install required package.
```

Comment:
You moved database documentation in another markdown file but it is not visible on the readme how to configure mysql or postgresql storage. Please add links to the specific documentations (something like "if you want to use mysql, please look at specific documentation - <here link to the documentation>)

Same thing for postgresql
Same thing for apache/wsgi configuration. 
Same thing for configuration parameters explanation